test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9600,6 +9713,51 @@ def import_hook(self, name, body, level=logging.INFO):
                         body + '---')
         return True
 
+    def import_hook_config(self, name, body, level=logging.INFO):
+        """
+        Helper function to import hook config by name.
+        The hook must have been created prior to calling this
+        function.
+
+        :param name: The name of the hook to import body to
+        :type name: str
+        :param body: The body of the hook config as a string.
+        :type body: str
+        :returns: True on success.
+        :raises: PbsManagerError
+        """
+        # sync_mom_hookfiles_timeout is 15min by default
+        # Setting it to lower value to avoid the race condition at hook copy
+        srv_stat = self.status(SERVER, 'sync_mom_hookfiles_timeout')
+        try:
+            sync_val = srv_stat[0]['sync_mom_hookfiles_timeout']
+        except BaseException:
+            self.logger.info("Setting sync_mom_hookfiles_timeout to 15s")
+            self.manager(MGR_CMD_SET, SERVER,
+                         {"sync_mom_hookfiles_timeout": 15})
+
+        fn = self.du.create_temp_file(body=body)
+
+        if not self._is_local:
+            tmpdir = self.du.get_tempdir(self.hostname)
+            rfile = os.path.join(tmpdir, os.path.basename(fn))
+            self.du.run_copy(self.hostname, fn, rfile)
+        else:
+            rfile = fn
+
+        a = {'content-type': 'application/x-config',
+             'content-encoding': 'default',
+             'input-file': rfile}
+        self.manager(MGR_CMD_IMPORT, HOOK, a, name)
+
+        os.remove(rfile)
```

Comment:
you can just use self.du.rm(self.hostname, rfile) for local as well, then you won't need the if condition below

Reply:
Function is no longer required since i have moved to qmgr command for getting hooks output and importing it